dev-scheme/guile/guile1-1.8.8.recipe

```diff
@@ -0,0 +1,155 @@
+	lib:libguile_srfi_srfi_13_14_v_3$secondaryArchSuffix = 3.0.1
+	lib:libguile_srfi_srfi_4_v_3$secondaryArchSuffix = 3.0.1
+	lib:libguile_srfi_srfi_60_v_2$secondaryArchSuffix = 2.0.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libgc$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libunistring$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	guile1${secondaryArchSuffix}_tools$secondaryArchSuffix = $portVersionCompat
+	cmd:guile$commandSuffix = $portVersion
+	cmd:guile_snarf$commandSuffix = $portVersion
+	cmd:guile_tools$commandSuffix = $portVersion
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	guile1$secondaryArchSuffix == $portVersion base
+	lib:libgmp$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	guile1${secondaryArchSuffix}_devel = $portVersionCompat
+	cmd:guile_config$commandSuffix = $portVersion
+	devel:libguile$secondaryArchSuffix = $libguileVersionCompat
+	devel:libguilereadline_v_17$secondaryArchSuffix = $libguileVersionCompat
+	devel:libguile_srfi_srfi_1_v_3$secondaryArchSuffix = 3.0.2
+	devel:libguile_srfi_srfi_13_14_v_3$secondaryArchSuffix = 3.0.1
+	devel:libguile_srfi_srfi_4_v_3$secondaryArchSuffix = 3.0.1
+	devel:libguile_srfi_srfi_60_v_2$secondaryArchSuffix = 2.0.2
+	"
+REQUIRES_devel="
+	guile1$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libffi$secondaryArchSuffix
+	devel:libgc$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libunistring$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gperf
+	cmd:grep
+	cmd:make
+	cmd:ld
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+defineDebugInfoPackage guile1$secondaryArchSuffix \
+	"$libDir"/libguile.so.$libguileVersion
+
+BUILD()
+{
+	CPPFLAGS="-D_BSD_SOURCE" LIBS="-lbsd -lnetwork" \
+	runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir \
+		--with-threads --disable-error-on-warning
+
+	# No MADV_DONTNEED, yet
+	sed --in-place '/HAVE_SYS_MMAN_H/d' config.h
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f "$libDir"/charset.alias
+	rm -f "$libDir"/lib*.la
+	rm -f "$libDir"/guile/$portVers/extensions/*.la
+	rm -f "$libDir"/guile/$portVers/extensions/*.a
+
+	prepareInstalledDevelLib libguile
+	prepareInstalledDevelLib libguilereadline-v-17
+	prepareInstalledDevelLib libguile-srfi-srfi-1-v-3
+	prepareInstalledDevelLib libguile-srfi-srfi-13-14-v-3
+	prepareInstalledDevelLib libguile-srfi-srfi-4-v-3
+	prepareInstalledDevelLib libguile-srfi-srfi-60-v-2
```

Comment:
These should be combined with "prepareInstalledDevelLibs"

Reply:
Ok. I'm fixing and checking right now. The splitting of the package into base and tools  as suggested by @extrowerk  solves the conflicts with guile 2.2. So I think I'm done, unless you see some other packaging problems. I've also updated accordingly the PR #4812 for TeXmacs.

Reply:
Hope I'll get some time to check the things, but so far it looks OK

Reply:
No hurry. I just wanted to be sure was on the radar. I would like to announce soon that TeXmacs is officially in Haiku. :)